plugins/lando/README.md

```diff
@@ -0,0 +1,30 @@
+
+# Lando ZSH (lando-zsh)
+This plugin adds aliases for using various languages and frameworks with [Lando](https://docs.lando.dev/basics/) for Docker. It will only run within lando-driven project directories.
+
+#### USAGE:
+```
+plugins=(... lando)
+```
+
+#### ALIASES:
+| Alias | Description |
+|:-:|:-:|
+| `artisan`  | `lando artisan`  |
+| `composer`  | `lando composer`  |
+| `drush`  | `lando drush`  |
+| `gulp`  | `lando gulp`  |
+| `npm`  | `lando npm`  |
+| `wp`  | `lando wp`  |
+| `yarn`  | `lando yarn`  |
```

Comment:
In rendered Markdown this looks fine, but it's more pleasant for the eyes to format the table so that the lines coincide:
```md
| Alias      | Description      |
|:----------:|:----------------:|
| `artisan`  | `lando artisan`  |
| `composer` | `lando composer` |
| `drush`    | `lando drush`    |
| `gulp`     | `lando gulp`     |
| `npm`      | `lando npm`      |
| `wp`       | `lando wp`       |
| `yarn`     | `lando yarn`     |
```

Reply:
Agreed!